package.json

```diff
@@ -85,6 +85,7 @@
   },
   "resolutions": {
     "**/@agoric/xsnap": "0.14.3-dev-9f085d3.0",
-    "**/@agoric/time": "0.3.3-dev-97cb715.0"
+    "**/@agoric/time": "0.3.3-dev-97cb715.0",
```

Comment:
could these all be `ut15` now?

Reply:
I meant 'u15' which is the version segment for upgrade-15 release. I see the new dep is using that.

Reply:
Ohh. Thanks for clarifying. :) Updated